.requirements

```diff
@@ -10,5 +10,5 @@ RESTY_LMDB_VERSION=1.0.0
 RESTY_EVENTS_VERSION=0.1.3
 ATC_ROUTER_VERSION=1.0.0
 LIBYAML_VERSION=0.2.5
-KONG_BUILD_TOOLS_VERSION=4.33.9
+KONG_BUILD_TOOLS_VERSION=4.33.10
```

Comment:
Do we need to make another release of KONG_BUILD_TOOLS?  I made 4.33.10 in the morning.

Reply:
```suggestion
KONG_BUILD_TOOLS_VERSION=4.33.11
```